src/DataFactory/DataFactory/ChangeLog.md

```diff
@@ -18,6 +18,8 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Added support for Iceberg format as a sink.
```

Comment:
Do you have the help document update?

Reply:
No, ADF only update sdk for powershell, nothing change in cmd

Reply:
> No, ADF only update sdk for powershell, nothing change in cmd

@Jingshu918 Please confirm you module is not going to release a new version after the merge is merged

Reply:
I don't know what you mean "module is not going to release a new version", After the PR merged Powershell should release a new version for DataFactory, for such change we have released many years.

https://github.com/Azure/azure-powershell/pull/25825
https://www.powershellgallery.com/packages/Az.DataFactory/1.18.7

Reply:
@Jingshu918 Sorry, please ignore the previous comment. I may comment in the wrong PR.
The review comments for you PR is
Please refer to https://github.com/Azure/azure-powershell/blob/main/documentation/development-docs/help-generation.md to update your help doc of your module

Reply:
The link is for powershell autorest track 2, this is not we are using. And we are not updating our module, why need update the docs?

Reply:
We are not updating our modules, so we need not do that.

Reply:
> We are not updating our modules, so we need not do that.

The help doc should be consistent with the source codes and currently the cmdlet help docs are already out of date.

Reply:
I don't think it is out of data in DataFactory. DataFactory maybe a special Service from others.
For example, we only change the definitionFile which customers can know from other places and it did not need customer change the modules and docs.

Set-AzDataFactoryV2Dataset
   [-Name] <String>
   [-DefinitionFile] <String>
   [-ResourceGroupName] <String>
   [-DataFactoryName] <String>
   [-Force]
   [-DefaultProfile <IAzureContextContainer>]
   [-WhatIf]
   [-Confirm]
   [<CommonParameters>]
   
   https://learn.microsoft.com/en-us/powershell/module/az.datafactory/set-azdatafactoryv2dataset?view=azps-12.3.0
   
   https://learn.microsoft.com/en-us/azure/data-factory/